90.md

```diff
@@ -67,6 +67,37 @@ All tags are optional.
 * `relays`: List of relays where Service Providers SHOULD publish responses to
 * `p`: Service Providers the customer is interested in. Other SPs MIGHT still choose to process the job
 
+## Encrypted Params 
+
+If the user wants to keep the input parameters a secret, they can encrypt the `i` and `param` tags with the service provider's 'p' tag and add it to the content field. The user also indicates whether the output should be encrypted or not as one of the parameters.
+
+```json
+[
+    [ "i", "what is the capital of France? ", "prompt" ],
```

Comment:
this should be `text` instead of `prompt`?

Reply:
Yes. Here's this is just an example. Actual structure will be defined in the DVM/kinds repo? 

`prompt` is used universally in the AI APIs for both text and image. But if we think DVM is more than AI, then we we can change it to text.

Reply:
My understanding is that as per https://github.com/nostr-protocol/nips/blob/vending-machine/90.md, `<input-type>` must be one of `text`, `url`, `job`, `event`

Reply:
yeah, @cj-chua is right, the third value should be `text` as that defines how the input should be interpreted (i.e. it's text, it's not a job id or an event ID that should be fetched, `text` indicates the input should be used as-is)

Reply:
agree and changed.